### ISSUE TITLE
Staging Review finding: Warning alert unlikely to be announced by screen readers

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#82854]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Pension benefits
**Product:** 527EZ
**Feature:** Update multi-response (single page list loop) pattern

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/employment/current/history](https://staging.va.gov/pension/apply-for-veteran-pension-form-21p-527ez/employment/current/history) when the maximum number of jobs has been entered, the "You've entered the maximum number of items allowed" message is likely to be missed by screen reader users.

When a screen reader user moves through a form, they don't typically have their screen reader announce all of the text on the page as they might on an informational page. Instead, they tab through the interactive elements on the page (buttons, links, form inputs) and hear their screen reader announce the labels programmatically associated with those elements.

The "You've entered the maximum..." message isn't programmatically associated with anything else, and blind users won't get the visual cue that something new has appeared on the page once they've added their second job. This could lead to confusion -- why isn't the Add another button there anymore?

## Recommended action

There are a few ways to solve this, but I think the lowest-effort and most reliable option might be to add helper text to the `legend` telling users up front what the limit is. Something like:
```
<fieldset>
	<legend>
		<h3>Current employment</h3>
		<span>Enter all your current jobs</span>
		<span>You may enter up to two jobs.</span>
	</legend>
	[cards]
</fieldset>
```

The warning alert once the maximum has been entered can be kept as-is. It's not exactly duplicative of the suggested helper text, but it's close enough that I wouldn't consider it a requirement that the warning be announced.

## References

- **Accessibility Defect Severity:** [2: High. Should be fixed in 1-2 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-2)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by echin393** [comment]
After reviewing our form, I've noticed that instructional text tends to live as a part of the sub-heading (h4), or as a part of a question label if preceding radio buttons. 

In this case, I'd suggest changing the current h4 heading from `Enter all your current jobs` to `Enter up to two of your current jobs`

<img width="1238" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMHpYQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--b7ed56b95532d118eb9e74f76833a3e627035992/image.png" alt="image.png" />

<img width="1230" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMHZYQkE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--2787fe98312bb6e4dcba9d78a401aff85f8b7d92/image.png" alt="image.png" />

**Comment by aplatt-coforma** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/benefits-pension-64e775aaa6b1ca1ed49b2ede/issues/department-of-veterans-affairs/va.gov-team/82854?planning-poker) with Zenhub @ToddWebDev @TaiWilkin
